www/class/centreonUtils.class.php

```diff
@@ -330,7 +330,7 @@ public static function escapeSecure(
     ) {
         switch ($escapeMethod) {
             case self::ESCAPE_LEGACY_METHOD:
-                return preg_replace("/<script.*?\/script>/s", "", $stringToEscape);
+                return self::escapeAllExceptSelectedTags($stringToEscape, ['script', 'input']);
```

Comment:
` public static function escapeAllExceptSelectedTags(
        $stringToEscape,
        $tagsNotToEscape = []
    ) {`
@loiclau is right. This method will escape everything except the script and input tags...